### ISSUE TITLE
Error on paste or new open file

### ISSUE THREAD

**Issue by exdeniz** [AtomLinter/linter-stylint#48]
I continuously see errors:

```
Error: Column start (30) greater than line length (6)
    at Object.rangeFromLineNumber (/Users/exdeniz/.atom/packages/linter-stylint/node_modules/atom-linter/lib/index.js:172:11)
    at /Users/exdeniz/.atom/packages/linter-stylint/lib/init.js:117:28
```

It fixed untabify all, but paste from another file the error appears again.

**Comment by phyllisstein** [comment]
I'm seeing the same thing. The error appears to be more or less stochastic, but I can consistently reproduce it by editing [the file in this Gist](https://gist.github.com/phyllisstein/4e4b09236397272c7c1f0328b0029e68). Would be grateful for any suggestions from the maintainers.

![screen recording 2016-08-12 at 05 26 am](https://cloud.githubusercontent.com/assets/2207980/17618389/7e4c2764-604d-11e6-954a-6be0ac1b8c20.gif)

**Comment by Arcanemagus** [comment]
This is almost always a bug in the linter (`stylint` in this case). I'll try to reproduce it locally.

**Comment by Arcanemagus** [comment]
This package doesn't seem to be working at all... it looks like the format for the results changed even though the `stylint` version hasn't, so it's completely broken right now 😮.

**Comment by Arcanemagus** [comment]
Ha, the specs only test the first error, and it would be correct, it's subsequent errors that are broken.

**Comment by Arcanemagus** [comment]
Wow, how does a linter behave this badly:
![image](https://cloud.githubusercontent.com/assets/427137/18568337/a057d550-7b51-11e6-99ce-da7d69c7cbf5.png)

If you define a config file it has an _entirely_ different output vs. when you don't. So this linter is working currently if you specify a configuration file, but broken for all but the first message if you don't.

**Comment by Arcanemagus** [comment]
I've fixed a race condition that could cause this in #63, and updated the regex to work with both output forms in #62.

**Comment by Arcanemagus** [comment]
And all the fixes are pushed out in v2.2.3, if you see anything after updating please file a new issue, thanks!

**Comment by phyllisstein** [comment]
Thanks for the fix!
